traffic_ops/traffic_ops_golang/user/current.go

```diff
@@ -80,3 +190,197 @@ WHERE u.id=$1
 	u.LocalUser = util.BoolPtr(localPassword.Valid)
 	return u, nil
 }
+
+func ReplaceCurrent(w http.ResponseWriter, r *http.Request) {
+	inf, userErr, sysErr, errCode := api.NewInfo(r, nil, nil)
+	tx := inf.Tx.Tx
+	if userErr != nil || sysErr != nil {
+		api.HandleErr(w, r, tx, errCode, userErr, sysErr)
+		return
+	}
+	defer inf.Close()
+
+	var userRequest tc.CurrentUserUpdateRequest
+	if err := json.NewDecoder(r.Body).Decode(&userRequest); err != nil {
+		errCode = http.StatusBadRequest
+		userErr = fmt.Errorf("Couldn't parse request: %v", err)
+		api.HandleErr(w, r, tx, errCode, userErr, nil)
+		return
+	}
+
+	user, exists, err := dbhelpers.GetUserByID(inf.User.ID, tx)
+	if err != nil {
+		sysErr = fmt.Errorf("Getting user by ID %d: %v", inf.User.ID, err)
+		errCode = http.StatusInternalServerError
+		api.HandleErr(w, r, tx, errCode, nil, sysErr)
+		return
+	} else if !exists {
```

Comment:
`else` is redundant here since the `if` block aways returns.

Reply:
That's true. I don't really think it's a big deal either way, but I can change it if you want.

Reply:
Which ever way you think the code is simpler and most clear. I personally think removing the `else` reduces the cognitive load of reading the code.